src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -61,11 +61,13 @@ const populateUISchemaForDisplay = (uiSchema, ionField) => {
   if (display.inputType === 'radio') {
     uiSchema.options = display.options;
   } else if (display.inputType === 'select') {
-    uiSchema.wide = true;
-    //it will create a placeholder for dropdowns, by default it will show 'Select an Option'
-    uiSchema.options = Object.assign({'': ''}, ionOptionsToUiOptions(display.options));
-  } else if (display.inputType === 'select_country_code') {
-    Object.assign(uiSchema, countryUISchema);
+    if(!_.isEmpty(display.format) && display.format === 'country-code'){
+      Object.assign(uiSchema, countryUISchema);
```

Comment:
nit: wondering if we can do something like this
<img width="894" alt="image" src="https://user-images.githubusercontent.com/90656212/157505256-3377143d-7a88-417f-93f1-9c9f0d119c2f.png">

Reply:
good suggestion. maybe we can implement this when we decide to do the same pattern for timezone and language code. thanks!